tests/tools/CMakeLists.txt

```diff
@@ -4,4 +4,7 @@
 if (OE_SGX)
     add_subdirectory(oecert)
     add_subdirectory(oecertdump)
+if (NOT WIN32)
```

Comment:
Can you add a # comment saying why this is in a NOT WIN32 condition?

Reply:
done